src/test/test_layout.py

```diff
@@ -0,0 +1,9 @@
+from common.layout import Layout
```

Comment:
All test cases should be implemented in the test environment. Simple tests cases/files like these can be created and  executed in your local repo, but removed before attempting to merge changes.